00YZ-au-frequency-plan-choice.md

```diff
@@ -0,0 +1,165 @@
+
+Since this will require the change of chain vars the proposal would be to revert this as part of the migration to the new routing infrastructure and LNS implementation post the Solana chain migration.
+
+Current device users will have to revert changes to their Sensors. Since the number of devices and data usage has been dropping past 17th November this should be done as a matter of urgency as soon as the chain migration has been completed and when manufacturers are required to switch over to the new gateway-rs implementation switching from LNS State channels to the NetID based routing infrastructure (coupled with the Chirpstack LNS switch).
+
+The following documentation will need to be modified.
+
+- [Frequencies on the Helium Network](https://docs.helium.com/LoRawan-on-helium/frequency-plans)
+- [Frequencies by Region on the Helium Network](https://docs.helium.com/LoRawan-on-helium/region-plans)
+
+The implementation procedure is the reversal of the change made on November 17th 2022 and is reversible in the same way.
+
+The impact should be dramatically less since this would reinstate the plan that was in place for the years leading up to November 17th and was extensively tested.
+
+## Success Metrics
+
+1. Network Coverage: improved network coverage in the region is measured by monitoring the number of new gateways deployed in the region, as well as the number of devices connecting to the network. An increase in these numbers would indicate a successful selection of the channel plan and improved network coverage.
+2. Device Compatibility: increased device compatibility is measured by monitoring the number of devices that are now able to connect to the network following the selection of the channel plan. An increase in the number of compatible devices would indicate a successful selection of the channel plan and increased device compatibility.
+3. Adoption Rate: monitoring the adoption rate of the network in the region by tracking the number of new users and applications developed on the network following the selection of the channel plan. An increase in the adoption rate would indicate a successful selection of the channel plan and increased adoption of the network by businesses and organisations in the region.
+4. Improved Service Quality: monitoring the quality of service provided by the network by tracking the number of network failures or disruptions following the selection of the channel plan. A reduction in these failures or disruptions would indicate a successful selection of the channel plan and improved service quality for users of the network.
+5. Restore faith in Helium: We expect this change would restore confidence in investors who purchased Helium-related hardware and have suffered reduced performance since November. We expect a restoration of faith in the Helium governance system and less of a feeling that the Australian investors have been ignored.
```

Comment:
```suggestion
5. Restore faith in Helium: We expect this change would restore confidence in Hotspot owners who purchased Helium-related hardware and have suffered reduced performance since November. We expect a restoration of faith in the Helium governance system and less of a feeling that the Australian users have been ignored.
```